### ISSUE TITLE
libatomic.so.1 not found

### ISSUE THREAD

**Issue by BinhongLiu** [soedinglab/MMseqs2#795]
## Expected Behavior
Compile the specific version [MMseqs2 71dd32ec43e3ac4dabf111bbc4b124f1c66a85f1 (May 28, 2023)](https://github.com/soedinglab/MMseqs2/archive/71dd32ec43e3ac4dabf111bbc4b124f1c66a85f1.zip) under Linux

I encounter a similar error as  [here](https://github.com/soedinglab/MMseqs2/issues/480). And the mentioned package `libgcc-ng` has been installed.

## Steps to Reproduce (for bugs)
Please make sure to execute the reproduction steps with newly recreated and empty tmp folders.

`cd MMseqs2-71dd32ec43e3ac4dabf111bbc4b124f1c66a85f1
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=RELEASE -DCMAKE_INSTALL_PREFIX=. ..
make
make install 
export PATH=/home/liuhongbin/database/colabfoldMSA/MMseqs2-71dd32ec43e3ac4dabf111bbc4b124f1c66a85f1/build/bin/:$PATH`

no errors were observed here.

## MMseqs Output (for bugs)
error:

(colabfoldMSA) liuhongbin@node7:~/database/colabfoldMSA/MMseqs2-71dd32ec43e3ac4dabf111bbc4b124f1c66a85f1/build$ ./bin/mmseqs
./bin/mmseqs: error while loading shared libraries: libatomic.so.1: cannot open shared object file: No such file or directory

## Context
Providing context helps us come up with a solution and improve our documentation for the future.

## Your Environment
Include as many relevant details about the environment you experienced the bug in.
* Git commit used (The string after "MMseqs Version:" when you execute MMseqs without any parameters):
71dd32ec43e3ac4dabf111bbc4b124f1c66a85f1
* Which MMseqs version was used (Statically-compiled, self-compiled, Homebrew, etc.):

* For self-compiled and Homebrew: Compiler and Cmake versions used and their invocation:
g++ (GCC) 5.4.0
cmake version 3.28.1
* Server specifications (especially CPU support for AVX2/SSE and amount of system memory):
uname -a | grep x86_64
Linux node7 3.10.0-1160.11.1.el7.x86_64 #1 SMP Fri Dec 18 16:34:56 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
* Operating system and version:
CentOS Linux release 8 (Core)

* packages in current environment
`conda list
packages in environment at /home/liuhongbin/anaconda3/envs/colabfoldMSA:

Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
absl-py                   1.4.0                    pypi_0    pypi
alphafold-colabfold       2.3.6                    pypi_0    pypi
appdirs                   1.4.4                    pypi_0    pypi
astunparse                1.6.3                    pypi_0    pypi
binutils                  2.40                 hdd6e379_0    conda-forge
binutils_impl_linux-64    2.40                 hf600244_0    conda-forge
binutils_linux-64         2.40                 hbdbef99_2    conda-forge
biopython                 1.82                     pypi_0    pypi
bzip2                     1.0.8                hd590300_5    conda-forge
c-compiler                1.7.0                hd590300_0    conda-forge
ca-certificates           2023.11.17           hbcca054_0    conda-forge
cachetools                5.3.2                    pypi_0    pypi
certifi                   2023.11.17               pypi_0    pypi
charset-normalizer        3.3.2                    pypi_0    pypi
chex                      0.1.85                   pypi_0    pypi
colabfold                 1.5.5                    pypi_0    pypi
contextlib2               21.6.0                   pypi_0    pypi
contourpy                 1.2.0                    pypi_0    pypi
cxx-compiler              1.7.0                h00ab1b0_0    conda-forge
cycler                    0.12.1                   pypi_0    pypi
dm-haiku                  0.0.10                   pypi_0    pypi
dm-tree                   0.1.8                    pypi_0    pypi
flatbuffers               23.5.26                  pypi_0    pypi
fonttools                 4.47.0                   pypi_0    pypi
gast                      0.5.4                    pypi_0    pypi
gcc                       12.3.0               h8d2909c_2    conda-forge
gcc_impl_linux-64         12.3.0               he2b93b0_3    conda-forge
gcc_linux-64              12.3.0               h76fc315_2    conda-forge
google-auth               2.26.1                   pypi_0    pypi
google-auth-oauthlib      1.2.0                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
grpcio                    1.60.0                   pypi_0    pypi
gxx                       12.3.0               h8d2909c_2    conda-forge
gxx_impl_linux-64         12.3.0               he2b93b0_3    conda-forge
gxx_linux-64              12.3.0               h8a814eb_2    conda-forge
h5py                      3.10.0                   pypi_0    pypi
idna                      3.6                      pypi_0    pypi
immutabledict             4.1.0                    pypi_0    pypi
importlib-metadata        4.13.0                   pypi_0    pypi
jax                       0.4.23                   pypi_0    pypi
jaxlib                    0.4.23                   pypi_0    pypi
jmp                       0.0.4                    pypi_0    pypi
keras                     2.15.0                   pypi_0    pypi
kernel-headers_linux-64   2.6.32              he073ed8_16    conda-forge
kiwisolver                1.4.5                    pypi_0    pypi
ld_impl_linux-64          2.40                 h41732ed_0    conda-forge
libclang                  16.0.6                   pypi_0    pypi
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-devel_linux-64     12.3.0             h8bca6fd_103    conda-forge
libgcc-ng                 13.2.0               h807b86a_3    conda-forge
libgomp                   13.2.0               h807b86a_3    conda-forge
libnsl                    2.0.1                hd590300_0    conda-forge
libsanitizer              12.3.0               h0f45ef3_3    conda-forge
libsqlite                 3.44.2               h2797004_0    conda-forge
libstdcxx-devel_linux-64  12.3.0             h8bca6fd_103    conda-forge
libstdcxx-ng              13.2.0               h7e041cc_3    conda-forge
libuuid                   2.38.1               h0b41bf4_0    conda-forge
libxcrypt                 4.4.36               hd590300_1    conda-forge
libzlib                   1.2.13               hd590300_5    conda-forge
markdown                  3.5.1                    pypi_0    pypi
markupsafe                2.1.3                    pypi_0    pypi
matplotlib                3.8.2                    pypi_0    pypi
ml-collections            0.1.1                    pypi_0    pypi
ml-dtypes                 0.2.0                    pypi_0    pypi
ncurses                   6.4                  h59595ed_2    conda-forge
numpy                     1.26.3                   pypi_0    pypi
oauthlib                  3.2.2                    pypi_0    pypi
openssl                   3.2.0                hd590300_1    conda-forge
opt-einsum                3.3.0                    pypi_0    pypi
packaging                 23.2                     pypi_0    pypi
pandas                    1.5.3                    pypi_0    pypi
pillow                    10.2.0                   pypi_0    pypi
pip                       23.3.2             pyhd8ed1ab_0    conda-forge
protobuf                  4.23.4                   pypi_0    pypi
py3dmol                   2.0.4                    pypi_0    pypi
pyasn1                    0.5.1                    pypi_0    pypi
pyasn1-modules            0.3.0                    pypi_0    pypi
pyparsing                 3.1.1                    pypi_0    pypi
python                    3.10.13         hd12c33a_1_cpython    conda-forge
python-dateutil           2.8.2                    pypi_0    pypi
pytz                      2023.3.post1             pypi_0    pypi
pyyaml                    6.0.1                    pypi_0    pypi
readline                  8.2                  h8228510_1    conda-forge
requests                  2.31.0                   pypi_0    pypi
requests-oauthlib         1.3.1                    pypi_0    pypi
rsa                       4.9                      pypi_0    pypi
scipy                     1.11.4                   pypi_0    pypi
setuptools                69.0.3             pyhd8ed1ab_0    conda-forge
six                       1.16.0                   pypi_0    pypi
sysroot_linux-64          2.12                he073ed8_16    conda-forge
tabulate                  0.9.0                    pypi_0    pypi
tensorboard               2.15.1                   pypi_0    pypi
tensorboard-data-server   0.7.2                    pypi_0    pypi
tensorflow-cpu            2.15.0.post1             pypi_0    pypi
tensorflow-estimator      2.15.0                   pypi_0    pypi
tensorflow-io-gcs-filesystem 0.35.0                   pypi_0    pypi
termcolor                 2.4.0                    pypi_0    pypi
tk                        8.6.13          noxft_h4845f30_101    conda-forge
toolz                     0.12.0                   pypi_0    pypi
tqdm                      4.66.1                   pypi_0    pypi
typing-extensions         4.9.0                    pypi_0    pypi
tzdata                    2023d                h0c530f3_0    conda-forge
urllib3                   2.1.0                    pypi_0    pypi
werkzeug                  3.0.1                    pypi_0    pypi
wheel                     0.42.0             pyhd8ed1ab_0    conda-forge
wrapt                     1.14.1                   pypi_0    pypi
xz                        5.2.6                h166bdaf_0    conda-forge
zipp                      3.17.0                   pypi_0    pypi`

**Comment by milot-mirdita** [comment]
If you use conda, please make sure to also use the conda-compilers and not your system compiler.

before calling `cmake` please do:
```
export CC=path-to-conda-gcc
export CXX=path-to-conda-g++
```
